app/containers/unconnectedAuthorShow/index.tsx

```diff
@@ -100,13 +101,24 @@ class AuthorShow extends React.PureComponent<AuthorShowProps> {
           }}
         />
       );
+
+      guideContext = (
```

Comment:
Just declare it with `const`

Reply:
This part is the same as 'itsMeButton' `let` Why should I declare it differently from the above?

Reply:
sorry. my mistake. NVM.